setup/create-wp-project/src/basics/command-line.js

```diff
@@ -5,6 +5,27 @@ const { exec } = require('promisify-child-process');
 // RangeError [ERR_CHILD_PROCESS_STDIO_MAXBUFFER]: stderr maxBuffer length exceeded
 const cloneTimeout = 45000;
 
+const checkNodeVersion = async () => exec(`node -v`);
+const checkComposerVersion = async () => exec(`composer -v`);
+const checkWPCliVersion = async () => exec(`wp --info`);
+const checkGitVersion = async () => exec(`git --version`);
+
+const checkRequirements = async () => {
+  const nodeVersion = await checkNodeVersion();
+  if (nodeVersion.stderr !== '') process.exit(0);
+  
+  const composerVersion = await checkComposerVersion();
+  if (composerVersion.stderr !== '') process.exit(0);
```

Comment:
None of these should exit with the status `0`, as status code 0 indicates successful execution:

> A successful command returns a 0, while an unsuccessful one returns a non-zero value that usually can be interpreted as an error code. Well-behaved UNIX commands, programs, and utilities return a 0 exit code upon successful completion, though there are some exceptions.

(from https://tldp.org/LDP/abs/html/exit-status.html)

Other scripts, such as deploy agents or provisioning scripts could rely on `npx create-wp-project`, so we should take care that it plays nicely.

```suggestion
  if (composerVersion.stderr !== '') process.exit(1);
```